chia/full_node/mempool_manager.py

```diff
@@ -549,10 +550,10 @@ async def check_removals(self, removals: Dict[bytes32, CoinRecord]) -> Tuple[Opt
         # 5. If coins can be spent return list of unspents as we see them in local storage
         return None, []
 
-    def get_spendbundle(self, bundle_hash: bytes32) -> Optional[SpendBundle]:
+    def get_spendbundle(self, spend_bundle_id: bytes32) -> Optional[SpendBundle]:
```

Comment:
this changes the function signature. any call site naming this parameter will need to be changed as well. It's unrelated to the PR and non-trivial. Did you inspect all call sites of `get_spendbundle()`?